Expensify/k2-extension
# Run eslint --fix on the code

HELD on https://github.com/Expensify/k2-extension/pull/79

Fixes https://github.com/Expensify/Expensify/issues/193464

# Tests
1. Run `npm run web`
2. Verify the extension builds properly
